internal/pkg/service/appproxy/appconfig/appconfig.go

```diff
@@ -31,45 +34,43 @@ type cacheItem struct {
 	expiresAt time.Time
 }
 
-func NewSandboxesAPILoader(logger log.Logger, clock clock.Clock, baseURL string, token string) Loader {
+func NewSandboxesAPILoader(logger log.Logger, clock clock.Clock, client client.Client, baseURL string, token string) Loader {
 	return &sandboxesAPILoader{
 		logger: logger,
 		clock:  clock,
-		sender: client.New().WithBaseURL(baseURL).WithHeader("X-KBC-ManageApiToken", token),
+		sender: client.WithBaseURL(baseURL).WithHeader("X-KBC-ManageApiToken", token),
 		cache:  make(map[string]cacheItem),
 	}
 }
 
-const staleCacheFallbackDuration = time.Hour
-
-func (l *sandboxesAPILoader) LoadConfig(ctx context.Context, appID string) (AppProxyConfig, error) {
+func (l *sandboxesAPILoader) LoadConfig(ctx context.Context, appID string) (AppProxyConfig, bool, error) {
```

Comment:
At first glance, I don't know what `bool` means, it would be better to name the arguments, `modified bool` / `valid bool`, or at least add a comment to the method. `The method returns "true" if ...`.
​

Reply:
Can you please explain it? (In the next PR).